faithcomesbyhearing/dbp-web
# FCBHDBP-576 Refine Requests to Exclude Filesets Not Matching the Correct Testament

# Description
When an user tries to changes the bible using the top menu the filesets are not being filtered and so, it is trying to retrieve incorrect filesets that don’t match with the selected book. We must to avoid pulling filesets that doesn't match with correct testament.

## Related Issue

https://fullstacklabs.atlassian.net/browse/FCBHDBP-576

## How to Test
- You should go the path: bible/JPNCJV/MAT/1 and it must load the correct page and the console doesn't render errors.
- You should go the path: biible/en1esv/mat/1 and it must load the correct page and the console doesn't render errors.
- You should go the path: biible/jpncjv/rut/2 and it must load the correct page and the console doesn't render errors.
- You should go the path: biible/engweb/mat/2 and it must load the correct page and the console doesn't render errors.
- You should go the path: bible/FRAPDF/MAT/1, it must load the correct page, the console doesn't render errors and it doesn't render request with 404 code.
- You should go a specific Algeria bible and it must load the correct page as you see in the following .gif.

![Peek 2023-06-02 01-28](https://github.com/faithcomesbyhearing/dbp-web/assets/73488660/91409a63-c1a0-4a56-834f-1c9b2f16cad9)
